.github/workflows/3_LSP_unit_tests_publish.yml

```diff
@@ -0,0 +1,83 @@
+name: Unit tests for LSP and publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      extension_version:
+        description: 'VSCode version'
+        required: true
+      branch:
+        description: 'Branch to run run this workflow on'
+        required: true
+
+env:
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  tests:
+    name: Run unit tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps: 
+      - uses: actions/checkout@v2
+        with: 
+          ref: ${{ github.event.inputs.branch }}
+      - name: Install Dependencies in LSP
+        run: cd packages/language-server && npm install
+      - name: Compile LSP
+        run: cd packages/language-server && npm run build
+      - name: LSP unit tests
+        run: cd packages/language-server && npm test
+
+  bump:
+    name: Bump LSP version
+    needs: [tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      - name: Install Dependencies
+        run: npm install
+      - name: checkout master if necessary
+        if: endsWith(github.ref, '/master') == false
+        run: git checkout master
+      - name: Update scripts version file
+        run: npm run bump_lsp_version_in_scripts ${{ github.event.inputs.npm_channel }} ${{ github.event.inputs.extension_version }}
```

Comment:
`bump_lsp_version_in_scripts` does not seem to exist in package.json:
https://github.com/prisma/language-tools/blob/simplify_CI/package.json
Am I looking in the wrong file?

Reply:
Yes, that's a leftover, I removed this.